app/views/admin/deals/_form.html.erb

```diff
@@ -0,0 +1,63 @@
+<%= form_with(model: [:admin, deal]) do |form| %>
+  <% if deal.errors.any? %>
```

Comment:
let's move this errors part to a partial in shared/errors.html.erb so that same can be used across all the forms

Reply:
done